### ISSUE TITLE
Why does Progress Ring Visual Update use 3px thickness?

### ISSUE THREAD

**Issue by riverar** [microsoft/microsoft-ui-xaml#4946]
Pulling @mdtauk's question out of a now abandoned PR to ensure it doesn't get ignored.

@mdtauk asks:
> I'm curious about the move to using 3px as a thickness, considering all the guidance and documentation uses 4px as the base measurement.
> 
> Has that guidance changed, or is it just not deemed necessary for these progress controls, and other paddings and margins that have been changed?

@karenbtlai responds:
> Great question @mdtauk ! This update is simply according to design guidance, and I'm pretty curious as to why it's 3px rather than multiples of 4px. @chigy would you be able to explain this better?

_Originally posted in https://github.com/microsoft/microsoft-ui-xaml/issues/4810#issuecomment-819810662_

**Comment by mdtauk** [comment]
Thanks @riverar

**Comment by michael-hawker** [comment]
@ranjeshj yeah, concerned the #4810 PR was approved and merged before these concerns were addressed.

Like this [is a tenant of our design language](https://docs.microsoft.com/en-us/windows/uwp/design/basics/design-and-ui-intro#multiples-of-four) called out in the docs.

**Comment by mdtauk** [comment]
> Multiples of four
>The sizes, margins, and positions of UI elements should always be in multiples of 4 epx in your UWP apps.
>
>UWP scales across a range of devices with scaling plateaus of 100%, 125%, 150%, 175%, 200%, 225%, 250%, 300%, 350%, and 400%. The base unit is 4 because it can be scaled to these plateaus as a whole number (for example; 4 x 125% = 5, 4 x 150% = 6). Using multiples of four aligns all UI elements with whole pixels and ensures UI elements have crisp, sharp edges. (Note that text doesn't have this requirement; text can have any size and position.)
>

**Comment by StephenLPeters** [comment]
@chigy please take a look

**Comment by chigy** [comment]
@riverar , @mdtauk , @michael-hawker 
Thank you for extracting this issue.

So, the 4epx grid applies to the entire ring footprint. So we recommend when one is creating progress ring, the sizes are 4px increments. For example, one of the sizes we often use is 32epx.

As to 3epx lines, in this visual, using 4epx line is simply too thick and does not look visually appealing.
However rounded corner radius does not work for 2epx (need minimum of 3 to make round), thus this control uses 3epx lines.
The concept of sharp edges doesn't really apply to this round control.

**Comment by mdtauk** [comment]
> @riverar , @mdtauk , @michael-hawker 
> Thank you for extracting this issue.
> 
> So, the 4epx grid applies to the entire ring footprint. So we recommend when one is creating progress ring, the sizes are 4px increments. For example, one of the sizes we often use is 32epx.
> 
> As to 3epx lines, in this visual, using 4epx line is simply too thick and does not look visually appealing.
> However rounded corner radius does not work for 2epx (need minimum of 3 to make round), thus this control uses 3epx lines.
> The concept of sharp edges doesn't really apply to this round control.
>

The outer border should remain crisp with the overall control size falling onto the grid at various scales, but the inner border could potentially render with a blurry edge.

Hopefully it will be a theoretical issue, rather than a practical one, but it may be something to bare in mind.
